src/tv2-common/getSegment.ts

```diff
@@ -125,6 +126,10 @@ export async function getSegmentBase<ShowStyleConfig extends TV2ShowStyleConfig>
 		segment.isHidden = false
 	}
 
+	segment.definesShowStyleVariant = iNewsStory.cues.some((cue) =>
+		cue ? cue[0].toLowerCase().includes('showstylevariant') : false
```

Comment:
Depending on the linting setup, you can use optional chaining instead and disregard the `false` literal:
```typescript
segment.definesShowStyleVariant = iNewsStory.cues.some(cue => cue?.[0].toLowerCase().includes('showstylevariant'))
```

Reply:
Changed to use optional chaining!